Packager.hx

```diff
@@ -179,50 +178,36 @@ class Packager {
 
 
     private static function createPhpPackage(classes: Array<String>): Void {
```

Comment:
The code that has been removed is already autogenerated by Haxe when creating the PHP package.